### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by nicolyoshikawa**
completed delete image for a review, completed delete image for a spot

### CODE REVIEW

File: backend/routes/api/reviews.js [hunk 1]
```diff
@@ -155,4 +161,38 @@ router.delete('/:id', requireAuth, async (req, res, next) => {
...
+        const err = new Error("Review couldn't be found");
+        err.status = 404;
+        return next(err);
+    }
+    if(userId !== reviewByID.userId){
+        const err = new Error("Forbidden");
+        err.status = 403;
+        return next(err);
+    }
+
```
Comment (E-F-III) [inline backend/routes/api/reviews.js:181]: May have a work around in deleting an image of a review that doesn't belong to current user

example : 
- review queried for by the id provided by the params may belong to the current user
- image queried for by the imageId provided by the params isnt associated with the review queried for ( reviewId on the image record doesn't match the review id provided

File: backend/routes/api/spots.js [hunk 2]
```diff
@@ -324,29 +324,37 @@ router.delete('/:id', requireAuth, async (req, res, next) => {
...
+        err.status = 404;
+        return next(err);
+    }
+    if(loggedInUser !== spotByID.ownerId){
+        const err = new Error("Forbidden");
+        err.status = 403;
+        return next(err);
+    }
+
+    const imageByID = await SpotImage.findByPk(imageId);
```
Comment (E-F-III) [inline backend/routes/api/spots.js:344]: May have a work around in deleting an image of a review that doesn't belong to current user

example : 
- spot queried for by the id provided by the params may belong to the current user
- image queried for by the imageId provided by the params isnt associated with the spot queried for ( spotId on the image record doesn't match the spot id provided

**Review by E-F-III — COMMENTED** [review]
Delete image concerns

File: backend/routes/api/reviews.js (hunk 1 above)
Comment (nicolyoshikawa) [reply backend/routes/api/reviews.js:181]: @E-F-III added where clause to grab the image belongs to the review

File: backend/routes/api/spots.js (hunk 2 above)
Comment (nicolyoshikawa) [reply backend/routes/api/spots.js]: @E-F-III dded where clause to grab the image belongs to the review

**Pull request merged**